assessment.rb

```diff
@@ -3,10 +3,37 @@
 
 # question 2:
 
+people = Person.create surname: 'Bishop', given_name: 'Caren', gender: 'f', height: 63, weight: 132, dob: 1943-09-26, eye_color: 'Brown', hair_color: 'Black'
+p people
+
+
 # question 3:
 
+namespace :db do
+  namespace :populate do
+desc 'Fill the people table with example data'
+  task people: :environment do
+    Person.transaction do
+      CSV.foreach(Rails.root + 'data/licensees.csv',
+       headers: true) do | person_row |
+        person = person_row.to_hash
+        Person.create(person)
+      end
+    end
+  end
+end
+end
+
+
 # question 4:
 
 # question 5:
 
+shorter = Person.find_by surname: 'Escobar', 'Whaley' given_name: 'Marilynn', 'Chris'
+
+shorter.height -= 1
+
+shorter.save!
+
+
```

Comment:
A good thought, but this won't work. `.height` is only available on an instance of the class not an 'ActiveRecord Relation' (i.e. a collection of instances). Additionally, your query structure is off. 
`surname: ['Escobar, 'Whaley']` is the right way to go.